src/99_appendices/03_hed.md

```diff
@@ -14,18 +14,18 @@ sub-control01/
         sub-control01_task-emoface_events.tsv
 ```
 ```
-onset duration  trial_type  HED
+onset duration  event_type  HED
 1.2 0.6 fixationCross Event/Category/Experimental stimulus, Event/Label/CrossFix, Event/Description/A cross appears at screen center to serve as a fixation point, Sensory presentation/Visual, Item/Object/2D Shape/Cross, Attribute/Visual/Fixation point, Attribute/Visual/Rendering type/Screen, Attribute/Location/Screen/Center
 5.6 0.008 target  Event/Label/Target image, Event/Description/A white airplane as the RSVP target superimposed on a satellite image is displayed., Event/Category/Experimental stimulus, (Item/Object/Vehicle/Aircraft/Airplane, Participant/Effect/Cognitive/Target, Sensory presentation/Visual/Rendering type/Screen/2D), (Item/Natural scene/Arial/Satellite, Sensory presentation/Visual/Rendering type/Screen/2D)
 500 0.008 nontarget Event/Label/Non-target image, Event/Description/A non-target image is displayed for about 8 milliseconds, Event/Category/Experimental stimulus, (Item/Natural scene/Arial/Satellite, Participant/Effect/Cognitive/Expected/Non-target, Sensory presentation/Visual/Rendering type/Screen/2D), Attribute/Onset
 ```
 
-Alternatively if the same HED tags apply to a group of events with the same `trial_type` they can be specified in the corresponding data dictionary (`_events.json` file) using the following syntax:
```

Comment:
I don't see the need to introduce a new prescribed column name. Why not use `trial_type`?

Reply:
Sorry evebt_type was a typo.  I think that trial_type is confusing --- there are usually several events in a trial (e.g., stimulus and response).  The example given of trial_type was of an experimental condition rather than something that would naturally correspond to an "event code" that would be coming off of an acquisition system.  I think it would be natural to have both.

Reply:
I would suggest opening a separate PR proposing the addition of `event_type` - it's going to be a much more controversial (it is potentially backward incompatible, it requires a clear definition in https://github.com/bids-standard/bids-specification/blob/master/src/04-modality-specific-files/03-task-events.md).  The issue of combining tags between JSON and TSV is much more straightforward and consensus will be easier to reach.

Reply:
Does trial_type have some special meaning in BIDS?  What I was proposing that the user could add any columns that they wanted to ( --- may I should call the column giraffes in the example?) and the idea would be that HED maps that were defined at various levels would be applied if they matched column names. I didn't mean to conflict with a BIDS standardized column.  It is not meant to be a prescribed column....

Reply:
`trial_type` is defined here: https://github.com/bids-standard/bids-specification/blob/master/src/04-modality-specific-files/03-task-events.md. I see your point and I guess this uncovers another lack of clarity in the current description - can one annotate any column from events.tsv with HED tags in events.json or only `trial_type`? If the answer is any column this needs to be clarified and could be part of this PR.